packages/flutter/lib/src/material/chip.dart

```diff
@@ -1179,6 +1189,20 @@ class _RawChipState extends State<RawChip> with MaterialStateMixin, TickerProvid
     if (!hasDeleteButton) {
       return null;
     }
+    final IconThemeData iconTheme = widget.iconTheme
+      ?? chipTheme.iconTheme
+      ?? theme.chipTheme.iconTheme
+      ?? const IconThemeData(size: _kDeleteIconSize);
+    final Color? effectiveDeleteIconColor = widget.deleteIconColor
```

Comment:
Wow. I get it, but wow.

Reply:
@HansMuller 
Looks like some tests are definitely failing in `Google Testing`. Could you please take a look?

Reply:
All the failures are related to the icon size change because now it respects `_kDeleteIconSize` instead of using the fallback size 24.0. I'm creating a fix in G3 to preserve the original change. Once that is merged, we can try to trigger the test here again:)

Reply:
Appreciate it!